src/views/Scenario/Scenario.js

```diff
@@ -228,6 +245,30 @@ const Scenario = () => {
     label: scenarioListLabel,
     validationStatus: scenarioValidationStatusLabels,
   };
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+  const lastRun = '2023-03-13T16:25:30.795486885Z';
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+
+  const hasScenarioBeenRun = useMemo(
+    () => currentScenario?.data?.lastRun !== null && currentScenario?.data?.state === 'Successful',
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [currentScenario?.data?.lastRun]
+  );
+  const isDashboardSync = useMemo(
+    () => new Date(currentScenario?.data?.lastUpdate) < new Date(lastRun),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [currentScenario?.data?.lastUpdate]
```

Comment:
`lastRun` is missing from the dependency array, was it done on purpose? (even though lastRun is currently hard-coded, it won't be hard-coded when the missing calls to the API will be added: we will probably need it in the dependency array then)

Reply:
The same, I thought it wasn't necessary because an update is always performed before a run but I'll put it in the array :+1: